react/Banner.js

```diff
@@ -38,8 +38,8 @@ class Banner extends Component {
   render() {
     const { image, page, targetParams } = this.props
     return (
-      <div>
-        <Link page={page} params={targetParams}>
+      <div className="flex items-center">
+        <Link className="mb0" page={page} params={targetParams}>
           <img src={image} />
```

Comment:
`img` should have `alt` attribute for a11y. Can you add it in this case?

Reply:
Bem lembrado. Tem que ter que adicionar a settings da imagem pra receber não so a URL como o texto de `alt`.